designs/virtualization/4.6/snapshots/snapshots.md

```diff
@@ -0,0 +1,62 @@
+---
+parent: Virtualization
+version: 4.6
+---
+
+# VM snapshots
+
+This item will cover updates to vm-snapshots
+- Creating a VM snapshot
+- Snapshots on the VM details page
+- Restoring a VM from snapshot
+
+## Creating a snapshot
+
+![Virtualization - VM snapshots tab](img/0-0.jpg)
+From the virtual machine details  page > snapshots tab, the user selects Take Snapshot”.
+
+![Virtualization - take snapshot](img/0-1.jpg)
+A modal will be presented, requesting for a snaoshot name.
+An autogenerated name combined with the VM name and the date of the snapshot, this will help in sorting the items on the table.
+
+![Virtualization - VM snapshot taken](img/0-2.jpg)
+On creation, the user returns to the VM details snapshot list (because the snapshot was taken from a modal)
+
+## VM snapshot details page
+
+![Virtualization - VM snapshot details page](img/1-0.jpg)
+Clicking on the VM snapshot name will land the user on the snapshot details page.
+At the moment, the current implementation of the IA (visible in the breadcrumb) is:
+'Virtual Machine Snapshot > Virtual Machine Snapshot Details'
+This should be changed in the future to"
+'Virtualization > virtual machines > virtual machine details > snapshots > snapshot details'
+
+## Snapshots on the VM dashboard
+
+![Virtualization - VM dashboard](img/2-0.jpg)
+Snapshots will be surfaced in the VM dashboard > Inventory, linked to the snapshots list
+
+## Snapshots for running VMs
+
+![Virtualization - Running VM](img/4-0.jpg)
```

Comment:
can you update this to use a PF alert
Also this modal doesn't really follow the description below it does it? The title should also align with the other modal where it says "Take snapshot" right?

Reply:
actually looking at the current implementation it doesn't allow a snapshot if the VM is running. So should this modal even be available?

Reply:
There is no modal today, the actions are disabled. This modal will be pushed to 4.x